modules/environment/aws/eks/main.tf

```diff
@@ -32,8 +32,8 @@ EOF
 
   map_roles_extra = var.enable_aws_code_services ? [
     {
-      rolearn  = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/aws-codebuild"
-      username = "user"
+      rolearn  = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/codebuild-role-lead"
```

Comment:
could we output this role arn from the `aws_iam_role. codebuild_role` resource in the code services module, instead of hardcoding it here?  or maybe pass the role name as a variable to both modules so its defined in one place?

Reply:
good call, I passed the output from the codeservices module into the eks module as a variable